packages/web/src/components/drawer/Drawer.tsx

```diff
@@ -175,7 +175,8 @@ const DraggableDrawer = ({
   // Handle the "controlled" component
   useEffect(() => {
     if (isOpen) {
-      open()
+      // Ensure that our drawer knows the height of its inner contents
+      setImmediate(open)
```

Comment:
According to the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate) this is non-standard. Is your intention to just run this in the next tick? If so, might get better to just `setTimeout(open, 0)`. I think it's likely that setImmediate just gets transpiled to a setTImeout anyway?

(A little reading tells me that the polyfills are a little more complicated and setTimeout is _slightly_ slower)

Or if you want to make sure this code runs _after_ the DOM elements are initialized, you could switch it to [`useLayoutEffect`](https://react.dev/reference/react/useLayoutEffect#measuring-layout-before-the-browser-repaints-the-screen)